src/core/mac/mac_types.hpp

```diff
@@ -959,6 +960,96 @@ class CslAccuracy
     uint8_t mUncertainty;
 };
 
+#if OPENTHREAD_CONFIG_WAKEUP_COORDINATOR_ENABLE
+/**
+ * Represents a wake-up request.
+ */
+class WakeupRequest : public otWakeupRequest
+{
+public:
+    /**
+     * Represents a wake-up request type.
+     */
+    enum Type : uint8_t
+    {
+        kTypeExtAddress    = OT_WAKEUP_TYPE_EXT_ADDRESS,
+        kTypeWakeupId      = OT_WAKEUP_TYPE_IDENTIFIER,
+        kTypeGroupWakeupId = OT_WAKEUP_TYPE_GROUP_IDENTIFIER,
+    };
+
+    /**
+     * Sets the wake-up request with an Extended Address.
+     *
+     * The type is also updated to indicate that the wake-up request type is kTypeExtAddress.
```

Comment:
```suggestion
     * The type is also updated to indicate that the wake-up request type is `kTypeExtAddress`.
```

Please update the other instances of kTypeExtAddress added in this file to be code font too.